2.4/root/usr/bin/run-mongod

```diff
@@ -46,8 +32,8 @@ function cleanup() {
     mongo_remove
   fi
   echo "=> Shutting down MongoDB server ..."
-  if [ -f "${MONGODB_PID_FILE}" ]; then
-    kill -2 $(cat ${MONGODB_PID_FILE})
+  if [[ -s ${MONGODB_DATADIR}/mongod.lock ]]; then
+    kill -2 $(cat ${MONGODB_DATADIR}/mongod.lock)
```

Comment:
>it's easier to review PRs that do one thing.
That change would be much easier to review in a PR of itself IMHO.

Yes. But it depends on graininess. If "every line change" is submited in separate PR, it really makes improvements harder and makes thinks slower.

However, there is a new PR #155